assets/js/blocks/price-filter/block.js

```diff
@@ -45,10 +45,10 @@ const PriceFilterBlock = ( { attributes } ) => {
 	const { showInputFields, showFilterButton } = attributes;
 	const minConstraint = isLoading
 		? undefined
-		: parseInt( results.min_price, 10 );
+		: Math.floor( parseInt( results.min_price, 10 ) / 10 ) * 10; // Nearest 10.
```

Comment:
Code style standards for WordPress js require the comment to be above the line it's related to.  So:

```js
const minConstraint = isLoading
		? undefined
		: parseInt( results.min_price, 10 );
        // Nearest 10
		: Math.floor( parseInt( results.min_price, 10 ) / 10 ) * 10;
```